src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/AbstractIndexWriter.java

```diff
@@ -224,19 +182,19 @@ protected void addClassInfo(TypeElement te, Content contentTree) {
                 ));
     }
 
-    protected void addDescription(SearchIndexItem sii, Content dlTree) {
-        String siiPath = pathToRoot.isEmpty() ? "" : pathToRoot.getPath() + "/";
-        siiPath += sii.getUrl();
-        HtmlTree labelLink = HtmlTree.A(siiPath, new StringContent(sii.getLabel()));
+    protected void addTagDescription(IndexItem item, Content dlTree) {
+        String itemPath = pathToRoot.isEmpty() ? "" : pathToRoot.getPath() + "/";
+        itemPath += item.getUrl();
+        HtmlTree labelLink = HtmlTree.A(itemPath, new StringContent(item.getLabel()));
         Content dt = HtmlTree.DT(HtmlTree.SPAN(HtmlStyle.searchTagLink, labelLink));
         dt.add(" - ");
-        dt.add(contents.getContent("doclet.Search_tag_in", sii.getHolder()));
+        dt.add(contents.getContent("doclet.Search_tag_in", item.getHolder()));
```

Comment:
Possible future RFE: `Search tag` is not a great term here, and doesn't really apply to tags like `{@systemProperty ...}` that exist for other reasons (e.g. to generate the System Properties page, and for which the entry in the interactive search is something of a secondary side-effect.   Given that the `IndexItem` contains the original `DocTree`, we could vary the description based on the `DocTree.Kind`.
That is not done here in this work, because it would likely require corresponding changes to tests.

Reply:
Also, what is the relationship between this content and the value in `IndexItem.getDescription()` ?